<launch>

  <node name="structure_learner" pkg="articulation_structure" type="structure_learner_srv" output="screen">
    <param name="filter_models" value="rigid prismatic rotational"/>
  </node>

  <node pkg="cmm_articulation" type="articulation_busybox.py" name="articulation_busybox" output="screen">
  </node>

  <node pkg="cmm_articulation" type="bb_poses_pub.py" name="bb_poses_pub" output="screen">
      <param name="poses_file_name" value="/media/sf_tmp/honda_cmm/full_busybox.json"/>
  </node>
  
  <node pkg="cmm_articulation" type="structure_subscriber.py" name="structure_sub" output="screen" >
      <param name="structure_file_name" value="/media/sf_tmp/honda_cmm/structure.json" />
  </node>
</launch>
